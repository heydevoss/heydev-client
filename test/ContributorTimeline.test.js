import { shallowMount } from '@vue/test-utils'
import ContributorTimeline from '@/components/user/ContributorTimeline'

describe('<Timeline/> Component', () => {
  const wrapper = shallowMount(ContributorTimeline)

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
