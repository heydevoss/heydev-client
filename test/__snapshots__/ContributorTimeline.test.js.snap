// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Timeline/> Component renders correctly 1`] = `
<div
  class="block"
>
  <br />
  <el-timeline>
    <el-timeline-item
      color="grey"
      icon="el-icon-s-promotion"
      size="large"
      timestamp="2019-06-12 20:46"
      type="primary"
    >
      
      New Pull Request on Tamburetei
    
    </el-timeline-item>
    <el-timeline-item
      color="grey"
      icon="el-icon-more"
      size="large"
      timestamp="2019-06-12 16:05"
    >
      
      Commented on IssueAi #48 issue
    
    </el-timeline-item>
    <el-timeline-item
      color="grey"
      icon="el-icon-star-on"
      size="large"
      timestamp="2019-06-11 13:56"
    >
      
      Star in Roadmap
    
    </el-timeline-item>
    <el-timeline-item
      color="grey"
      icon="el-icon-circle-plus-outline"
      size="large"
      timestamp="2019-06-10 19:36"
    >
      
      Commit on IssueAi
    
    </el-timeline-item>
  </el-timeline>
</div>
`;
